opal/corelib/number.rb

```diff
@@ -261,6 +261,16 @@ def **(other)
     end
   end
 
+  def ===(other)
+    %x{
+      if (other.$$is_number) {
+        return self.valueOf() === other.valueOf();
+      } else {
+        return #{self == other};
```

Comment:
NOTE: This line is mandatory in order to satisfy Ruby specs:
 * ["Fixnum calls 'other == self' if the given argument is not a Fixnum"](https://github.com/ruby/spec/blob/906bd11ebf8d75f8dba8824a986cf3d83ceda850/core/fixnum/shared/equal.rb#L13)
 * ["Float calls 'other == self' if coercion fails"](https://github.com/ruby/spec/blob/906bd11ebf8d75f8dba8824a986cf3d83ceda850/core/float/shared/equal.rb#L8)